packages/salesforcedx-vscode-lightning/test/deprecatedClassName.test.ts

```diff
@@ -11,20 +11,36 @@ import * as vscode from 'vscode';
 
 describe('SLDS Deprecated Class Name', () => {
   let res: vscode.Uri[];
+  let coreExtension: vscode.Extension<any>;
   let lightningExtension: vscode.Extension<any>;
 
   before(async () => {
     if (vscode.workspace.rootPath) {
+      coreExtension = vscode.extensions.getExtension(
+        'salesforce.salesforcedx-vscode-core'
+      ) as vscode.Extension<any>;
+
       lightningExtension = vscode.extensions.getExtension(
         'salesforce.salesforcedx-vscode-lightning'
       ) as vscode.Extension<any>;
     }
   });
 
+  it('coreExtension activation', async function() {
+    // tslint:disable-next-line:no-invalid-this
+    this.timeout(0);
```

Comment:
Broke down the original test into multiple tests. I also introduced this line which disables mocha's timeout restriction. This is needed since there's been an increase in the time needed to activate the core extension, which delays the lightning extension's activate.

Reply:
Based on appveyor, core extension activate is taking ~6800ms

Reply:
Would it be better to set a timeout that is longer than the core extension activation without disabling the timeout restriction altogether?

Reply:
It could be, that way we can catch if we make a change that significantly delays activation of core. Let me try it.